tests/create_command.t

```diff
@@ -25,9 +25,9 @@ Should succeed - create a secret in a new folder, with info about recipients add
   
   If the secret is a staging secret, its only recipient should be @everyone.
   
-Newly created secrets should have permissions 0o644
-  $ stat -c "%a" $PASSAGE_DIR/secrets/new/secret.age
-  644
+Newly created secrets should have permissions 0o644 or less
+  $ stat -c "%a" $PASSAGE_DIR/secrets/new/secret.age | more_deterministic_permissions
+  6xx
```

Comment:
i don't understand the need for these changes? Can you explain?

Also: `more_deterministic_permissions` allows for permissions `655`, `666` and `677` to pass the test, which is not what we want.

Reply:
The tests fail locally for me because the files get created with `0o600`:

```
 $ umask
0077
```

Not sure what the best approach for those tests should be (does it even make sense to test old unix permissions? hosts can have very complex “`setfacl`” settings, and filesystems can also override them)

I put that fix in a separate commit in case it's not wanted BTW.

_(I'd swear I had explained all of that in the PR description, must have forgot to paste or closed the tab too early or something :D )_

Reply:
that's odd that you get those permissions. It shouldn't happen, but testing permissions with stat is a mess because there are a few different implementations with different flags, so it's not really a universal solution.

Re acl, is there a way to "universally" check? These tests are run in isolation, so  real world systems can have different results, but I think we expect the users to have some control over their systems? I'm not sure if we can just handle the writes differently? This part of the code is a bit messy, I never had the opportunity to refactor it when I took over the project, but now it doesn't feel like we should make any big time investments, given the roadmap.

happy to hear any suggestions :)